### DISCUSSION TITLE
Configure search path for Active Directory?

### DISCUSSION THREAD

**Discussion by yooakim** [datalust/seq-tickets#1697] [Help] [answered]
I am helping a client of mine to setup a local SEQ instance. The VM is joined to their Active Directory and the user I am trying to use is logged in with domain credentials etc.

The problem I face is that each time I try to setup the Active Directory authentication it fails with a message saying it can't search AD.

I suspect the issue is because the LDAP path for the users is not the default in this particular AD.

How can I tell Seq to use another path for searching for users in Active Directory?

**Comment by nblumhardt** [comment, marked as answer]
Hi Joakim, this is most likely because the Seq service account is a limited machine account and therefore can't access AD. You can test this theory by switching the Seq service to `LocalSystem` and trying again.

Alternatively, you can try setting a switch that avoids LDAP in favor of SSPI whenever possible:

```
seq config set -k api.avoidLdap -v True
seq service restart
```

Let me know if either of the above helps, or if we need to dig in deeper.

Best regards,
Nick
